app/src/main/java/sergio/splashbaseviewer/utils/SplashBaseService.java

```diff
@@ -0,0 +1,21 @@
+package sergio.splashbaseviewer.utils;
+
+import retrofit2.Call;
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+import retrofit2.http.GET;
+import sergio.splashbaseviewer.utils.models.SplashBaseResponse;
+
+/**
+ * @author s.ruiz
+ */
+
+public interface SplashBaseService {
```

Comment:
Separate this please. Create an interface for declaration of calls, and create a class to manage the calls.